packages/ember-data/lib/adapters/rest_adapter.js

```diff
@@ -171,6 +171,58 @@ export default Adapter.extend({
   defaultSerializer: '-rest',
 
   /**
+    By default, the RESTAdapter will send the query params sorted alphabetically to the
+    server.
+
+    For example:
+    ```js
+      store.find('posts', {sort: 'price', category: 'pets'});
+    ```
```

Comment:
Do you mind adding newlines above and below this js example? The markdown parser doesn't seem to want to do code hi-lighting when the code block is so close to other text.

![screen shot 2015-01-25 at 11 36 35 pm](https://cloud.githubusercontent.com/assets/54056/5895383/39136690-a4eb-11e4-8ffa-114718156513.png)